### ISSUE TITLE
Contribution implies a pledge!?

### ISSUE THREAD

**Issue by thorwil** [ContributorCovenant/contributor_covenant#624]
This CoC contains "... we as contributors and maintainers pledge ...". A person who submitted a patch that got applied surely is a contributor, right? But a person may get a patch into a project:
- before the project adds this CoC
- being unaware of the presence of this CoC
- without reading it, even given awareness of the presence
In any case, this CoC claims that the contributor made a pledge, without the contributor ever having agreed to it. This language should be changed to remove this encroachment.

A lesser and far less practical alternative would be to add guidance that projects using it should demand explicit agreement before accepting any contribution, and that the CoC cannot be put in place without agreement of all extant contributors.

The name of the CoC, with "covenant" meaning "a formal agreement or promise between two or more people" has the same problem.

**Comment by elsandosgrande** [comment]
Good point. I'll notify Shane Peelar (GentooLTO; I will also need to add GentooLTO to adopters.csv) of this point of worry.
